docs/user/00-20-configure-serverless.md

```diff
@@ -0,0 +1,74 @@
+# Overview
+
+Serverless module has its own operator (`serverless-operator`). It watches `Serverless` Custom Resource and re-configure (reconcile) serverless workloads.
+
+Serverless CR becomes the API to configure serverless module. Use it to do the following:
+ - enable/disable internal docker registry
+ - configure external docker registry 
+ - override endpoint for traces collected by serverless functions
+ - override endpoint for eventing
+
+Default configuration of serverless module looks as follows:
+
+```yaml
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Serverless
+  name: serverless-sample
+spec:
+  dockerRegistry:
+    enableInternal: true
+```
+
+## Configure Docker Registry
+
+By default, Serverless uses PersistentVolume (PV) as the internal registry to store Docker images for Functions. The default storage size of a single volume is 20 GB. This internal registry is suitable for local development.
+
+If you use Serverless for production purposes, it is recommended that you use an external registry, such as Docker Hub, Google Container Registry (GCR), or Azure Container Registry (ACR).
+
+If you want serverless to use external docker registry, create a secret in `kyma-system` Namespace. Such a Secret must contain required data (**username**, **password**, **serverAddress**, and **registryAddress**):
+
+```bash
+kubectl create secret -n kyma-system generic my-registry-config --from-literal=username={your-docker-reg-username} --from-literal=password={your-docker-reg-password} --from-literal=serverAddress={your-docker-reg-server-url}  --from-literal=registryAddress={your-docker-reg-registry-url}
+```
+
+>**TIP:** In case of DockerHub, usually the Docker registry address is the same as the account name.
+
+Example:
+
+```bash
+kubectl create secret -n kyma-system generic my-registry-config --from-literal=username=kyma-rocks --from-literal=password=admin123 --from-literal=serverAddress=https://index.docker.io/v1/  --from-literal=registryAddress=kyma-rocks
```

Comment:
Is this important how type of secret we use? I see that in doc https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-08-switch-to-external-registry/ we use secret type `kubernetes.io/dockerconfigjson`.

Reply:
no its not important. The operator takes it and extracts fields and use those values as overrides